tools/main_for_build/extend_oem_bin.go

```diff
@@ -0,0 +1,32 @@
+package main
```

Comment:
Instead of putting this file in a main_for_build directory, you can either put it directly in the tools/ directory or in a tools/cmd/ directory.